core-web/libs/sdk/client/src/lib/editor/sdk-editor.ts

```diff
@@ -43,6 +43,29 @@ export function editContentlet<T>(contentlet: Contentlet<T>) {
     });
 }
 
+/**
+ * Reorders the menu based on the provided configuration.
+ *
+ * @param {ReorderMenuConfig} [config] - Optional configuration for reordering the menu.
+ * @param {number} [config.startLevel=1] - The starting level of the menu to reorder.
+ * @param {number} [config.depth=2] - The depth of the menu to reorder.
+ *
+ * This function constructs a URL for the reorder menu page with the specified
+ * start level and depth, and sends a message to the editor to perform the reorder action.
+ */
+export function reorderMenu(config?: ReorderMenuConfig): void {
+    const { startLevel = 1, depth = 2 } = config || {};
+
+    // This is the URL for the reorder menu page
+    // All params are hardcoded on the jsp, so here we just need to send the same URL
+    const reorderUrl = `/c/portal/layout?p_l_id=2df9f117-b140-44bf-93d7-5b10a36fb7f9&p_p_id=site-browser&p_p_action=1&p_p_state=maximized&_site_browser_struts_action=%2Fext%2Ffolders%2Forder_menu&startLevel=${startLevel}&depth=${depth}`;
```

Comment:
Why we are building the url here but then we do it again here `createReorderMenuURL`... can we just unify in one place?